src/taipy/gui/gui.py

```diff
@@ -1994,8 +1996,23 @@ def run(
             flask_log=app_config["flask_log"],
             run_in_thread=app_config["run_in_thread"],
             allow_unsafe_werkzeug=app_config["allow_unsafe_werkzeug"],
+            notebook_proxy=app_config["notebook_proxy"],
         )
 
+    def reload(self):
```

Comment:
Are you saying that, instead of stop()/run(), users should call 'reload()' when using Notebooks?

Reply:
that is what is on my mind. if you dont pass anything into the run function then it doesnt make any difference. But if you do, it will preserve the config and reloads everything